build_script_generator.go

```diff
@@ -121,6 +121,13 @@ func (g *webBuildScriptGenerator) Generate(ctx gocontext.Context, job Job) ([]by
 		u.User = nil
 	}
 
+	jp := job.Payload()
+	if jp != nil {
+		q := u.Query()
+		q.Add("job_id", fmt.Sprintf("%d", jp.Job.ID))
```

Comment:
Should this be `Set` instead of `Add`? It's only going to make a difference if the URL already has `job_id` for some reason, which I think is unlikely, but if it is set we'd effectively be sending an array instead of a single value.